templates/nextflow-efs-file-system.yaml

```diff
@@ -0,0 +1,145 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: EFS shared file system for ECS Tower Task containers
+
+Parameters:
+
+  VpcId:
+    Description: ID of VPC
+    Type: AWS::EC2::VPC::Id
+
+  SubnetId:
+    Description: The ID of the subnet to add the EFS mount target in.
+    Type: String
+
+  EcsSecurityGroupId:
+    Type: AWS::EC2::SecurityGroup::Id
+    Description: Security group ID for ECS cluster to grant database access
+
+  NextflowTowerConfigBucketArn:
+    Type: String
+    Description: ARN of the S3 bucket where the tower config is stored
+
+Resources:
+
+  FileSystem:
+    Type: AWS::EFS::FileSystem
+    Properties:
+      AvailabilityZoneName: us-east-1a
+      BackupPolicy:
+        Status: DISABLED
+      Encrypted: false
```

Comment:
not gonna do anything about this one?

Reply:
@zaro0508 Do you think I need to encrypt a bucket that only contains config files for Tower, i.e. not PHI?

Reply:
i guess not but @ahayden might think otherwise.  anyways it would be pretty easy just to set encryption using the  default key correct?  i think we only require AES encryption on s3 buckets so i'm guessing default key here is fine.

Reply:
@zaro0508 Ok, will do this in a follow-up PR later today